test/functionalTest/harnessFunctions.sh

```diff
@@ -918,6 +923,32 @@ function mongoCmd()
 
 
 
+# ------------------------------------------------------------------------------
+#
+# mongoCmd2 - like mongoCmd but showing all the output, not just the last line.
```

Comment:
Name should be more meaninful than mongoCmd2, eg. mongoCmdLong.

Reply:
Fixed in e0bb95f